Pieces/Piece.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Chess.Board;
+namespace Chess.Pieces
+{
+    abstract class Piece
+    {   
+        protected bool first_move = true;
+        public string name;
+        protected string piece_index;
+        protected ConsoleColor team_color;
+        public ConsoleColor Team_Color { get => team_color; }
```

Comment:
G24. G12, skoro używasz properties to używaj Automatic Properties, zmienna `team_color` z linijki 14 jest w takim razie nie potrzebna, 
```suggestion
        public TeamColor Color { public get; protected set; }
```

Reply:
Poprawiono